### ISSUE TITLE
When keyboard layout is Cyrillic, keyboard shortcuts do not work.

### ISSUE THREAD

**Issue by ELigoP** [johannesjo/super-productivity#3099]
### Your Environment

- Version used: 8.0.1 AppImage
- Operating System and version: Ubuntu 22.04
- Desktop Environment: gdm3 42.0

### Expected Behavior

Shift+A keyboard shortcut should add new task when in Russian keyboard layout.

### Current Behavior

Shift+A keyboard shortcut does nothing.

### Steps to Reproduce (for bugs)

1. Open the SuperProductivity app.
2. Change keyboard layout to Russian/Serbian etc.
3. Press Shift+A

### Can you reproduce this reliably?

Yes.

### Error Log (Desktop only)

No related errors and no errors at given time when I perform action.

**Comment by ELigoP** [comment]
https://stackoverflow.com/questions/70211837/keyboard-shortcuts-commands-on-non-latin-alphabet-keyboards-javascript - if preserving strings in form of "Shift+A" is important, then this might be way to go, `src/app/util/check-key-combo.ts` needs to be modified. Can do this later if I have some time...

**Comment by johannesjo** [comment]
That would be great! Thank you!

**Comment by ELigoP** [comment]
Yes, it is. Have tried a few things, haven't time/energy to finish yet.

**Comment by johannesjo** [comment]
No problem! Thanks for letting us know!

**Comment by Jagdfalke** [comment]
Hey @ELigoP . Are you still working on this?

**Comment by johannesjo** [comment]
Related https://github.com/johannesjo/super-productivity/pull/3659

**Comment by SatyrTheEarthling** [comment]
Hi, is this feature was rejected or already in prod? 
It's don't work for me in v11.
For me it's just a killer feature - without it the whole experience of using goes to nothing.

**Comment by johannesjo** [comment]
@SatyrTheEarthling I was under the impression that this has been addressed, but maybe that's not the case.

**Comment by johannesjo** [comment]
I think this should be fixed now?

**Comment by wafemand** [comment]
`Shift+a` for creating a task and `a` for creating a subtask are working now! Tested on Cyrillic and Georgian layouts.
Haven't tested other shortcuts though.

As I understand, the issue was fixed by @novikov1337danil in #5156 

Thank you!

**Comment by novikov1337danil** [comment]
> As I understand, the issue was fixed by [@novikov1337danil](https://github.com/novikov1337danil) in [#5156](https://github.com/johannesjo/super-productivity/pull/5156)

You're right, it's been fixed. 
However, now there is a difficult problem related to layouts other than QWERTY - https://github.com/johannesjo/super-productivity/issues/5298

@johannesjo this issue can be closed.

**Comment by johannesjo** [comment]
Thank you!
